codesandbox/codesandbox-client
# feat(trial): Add onboarding questions

## Todos

- [x] ~[Submit role and usage questions](https://linear.app/codesandbox/issue/XTD-190/save-answers-to-user-metadata).~

## How to test this

1. Go to the `/signin` page in an incognito window (or any other window where you're not logged in)
2. Open the browser console. paste the following code and press enter to execute:
```
localStorage.setItem('TOQ_DEBUG', 'ENABLED');
```
3. Refresh the page
4. You should now be able to see the onboarding questions

### Test the modal

1. Go to `/s` page in an incognito window (or any other window where you're not logged in)
2. Go to the second tab or any other tab with the sign in button
3. Open the browser console. paste the following code and press enter to execute:
```
localStorage.setItem('TOQ_DEBUG', 'ENABLED');
```
4. Press the sign in button
5. You should now be able to see the onboarding questions

**Note**: I wasn't able to properly test the whole flow, because of issues with redirects / auth happening on multiple different platforms and developer login opening on stream. We're only really able to tests styles at this point.

## Screenshots

<img width="1509" alt="image" src="https://user-images.githubusercontent.com/7533849/196981719-4fd2e6ed-3e9a-4750-a283-eca1fc152058.png">

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/7533849/196981793-b1acde51-2d74-4387-bc35-3ce5649312a0.png">
